spec/features/viewing_a_manual_spec.rb

```diff
@@ -128,23 +128,6 @@
     expect(page.status_code).to eq(404)
   end
 
-  scenario "visiting the obsolete prototype HMRC manual" do
-    content_store_does_not_have_item('/hmrc-internal-manuals/employment-income-manual')
-
-    visit_hmrc_manual "employment-income-manual"
-    expect_manual_title_to_be("This prototype of the Employment Income Manual is no longer available")
-    expect(page.status_code).to eq(200)
-  end
-
-  scenario "visiting a section in the obsolete prototype HMRC manual" do
-    content_store_does_not_have_item('/hmrc-internal-manuals/employment-income-manual')
-    content_store_does_not_have_item('/hmrc-internal-manuals/employment-income-manual/abc123')
-
-    visit_hmrc_manual_section "employment-income-manual", "abc123"
-    expect_manual_title_to_be("This prototype of the Employment Income Manual is no longer available")
-    expect(page.status_code).to eq(200)
-  end
-
   context "prototype HMRC manual has now been published for real" do
```

Comment:
It probably makes sense to get rid of this test too, it doesn't make sense with the fallback code gone.